block.go

```diff
@@ -130,10 +130,17 @@ type BlockReader interface {
 	Tombstones() (TombstoneReader, error)
 }
 
-// Appendable defines an entity to which data can be appended.
-type Appendable interface {
```

Comment:
Isn't there an incompatibility issue with removal of public interface?

Reply:
Yes, but we don't use this interface in tsdb itself. I am not sure we want to provide any guarantees about stability yet.

Reply:
Sure, but TSDB is a library. Anyone can depends on that including Prometheus itself. But I don't know, maybe the policy for TSDB is just to always vendor and pin to some version, so this is fine.